application/controllers/ReportController.php

```diff
@@ -47,6 +47,38 @@ public function indexAction()
         }
     }
 
+    public function cloneAction()
+    {
+        $this->assertPermission('reporting/reports');
+        $this->addTitleTab('Clone Report');
+
+        $values = [
+            'timeframe' => (string) $this->report->getTimeframe()->getId(),
+        ];
+
+        $reportlet = $this->report->getReportlets()[0];
+
+        $values['reportlet'] = $reportlet->getClass();
+
+        foreach ($reportlet->getConfig() as $name => $value) {
+            if ($name === 'name') {
+                $values[$name] = $value . ' Clone';
+                continue;
```

Comment:
Not to have a report name like `Host Reports Clone Clone Clone .....`
```suggestion
                if (preg_match('/(?:Clone)+(\d+)$/', $value, $matches)) {
                    $value = preg_replace('/\d+$/', ++$matches[1], $value);
                } else {
                    $value .= 'Clone 1';
                }

                $values[$name] = $value;

                continue;
```